### PR TITLE
Fine tune - Planning

### PR DESCRIPTION
**Pull request by Lennonka**
Fine tuning for Satellite 6.12 GA.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/doc-Planning_for_Project/topics/Provisioning_Concepts.adoc [hunk 1]
```diff
@@ -154,7 +154,7 @@ endif::[]
 
 === Kickstart
 You can use Kickstart to automate the installation process of a {ProjectName} or {SmartProxyServer} by creating a Kickstart file that contains all the information that is required for the installation.
-For more information about Kickstart, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/chap-kickstart-installations[Kickstart Installations] in the _{RHEL} 7 Installation Guide_.
+For more information about Kickstart, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_8_installation/performing_an_automated_installation_using_kickstart[Performing an automated installation using Kickstart] in the _{RHEL} 8 Performing an advanced RHEL 8 installation_.
```
Comment (adamlazik1) [inline guides/doc-Planning_for_Project/topics/Provisioning_Concepts.adoc:157]:
```suggestion
For more information about Kickstart, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_8_installation/performing_an_automated_installation_using_kickstart[Performing an automated installation using Kickstart] in _Performing an advanced RHEL 8 installation_.
```

File: guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc [hunk 2]
```diff
@@ -3,24 +3,24 @@
...
 +
 On large {Project} deployments, you can improve performance by configuring your {Project} with predefined tuning profiles.
 For more information, see {InstallingServerDocURL}tuning-with-predefined-profiles_{project-context}[Tuning {ProjectServer} with Predefined Profiles] in _{InstallingServerDocTitle}_.
 
-* For more information about installing {ProjectServer} from a disconnected network, see {MultiBaseURL}installing_satellite_server_from_a_disconnected_network/[_Installing {ProjectServer} from a Disconnected Network_] and {MultiBaseURL}installing_satellite_server_from_a_disconnected_network/preparing-environment-for-satellite-installation[Preparing your Environment for Installation] in the same guide.
+* For more information about installing {ProjectServer} in a disconnected network, see {InstallingDisconnectedDocURL}[_{InstallingServerDisconnectedDocTitle}_] and {InstallingDisconnectedDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in the same guide.
```
Comment (adamlazik1) [inline guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc:20]:
```suggestion
* For more information about installing {ProjectServer} in a disconnected network, see {InstallingDisconnectedDocURL}[_{InstallingServerDisconnectedDocTitle}_].
```
IMO the second link is not necessary.

**Review by adamlazik1 — COMMENTED** [review]
I can't confirm the deleted parts, so leaving rest of the review to @ekohl.

File: guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc:20]: Agreed

File: guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc [hunk 3]
```diff
@@ -3,24 +3,24 @@
...
-For an example implementation based on a sample customer scenario (specific to {ProjectXY}), see the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/articles/1585273[10 Steps to Build an SOE: How Red Hat Satellite 6 Supports Setting up a Standard Operating Environment].
+For an example implementation based on a sample customer scenario (specific to {Project}), see the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/articles/1585273[10 Steps to Build an SOE: How Red Hat Satellite 6 Supports Setting up a Standard Operating Environment].
 endif::[]
 
 ifdef::satellite[]
 [[sect-Satellite_Server_Configuration]]
 === {ProjectServer} Configuration
 
-The first step to a working {Project} infrastructure is installing an instance of {ProjectServer} on a dedicated {RHEL} 7 or dedicated {RHEL} 8 Server.
+The first step to a working {Project} infrastructure is installing an instance of {ProjectServer} on a dedicated {RHEL} 8 Server.
```
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc:13]: I'd say it should be `server`. Previously was already wrong, but for RHEL 7 there's a Server edition (and Workstation) but for RHEL 8 there's just a single edition. So It used to be `{RHEL} 7 Server` which referred to a product name. Now I think `on a dedicated {RHEL} 8` can look odd to a user so you could add a noun `server` to it. So lowercase it.

File: guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc [hunk 4]
```diff
@@ -258,13 +258,15 @@ The same guide contains information on configuring external authentication sourc
 === Additional Tasks
 
 This section provides a short overview of selected {Project} capabilities that can be used for automating certain tasks or extending the core usage of {Project}:
-
+////
```
Comment (ekohl) [inline guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc:261]: I'm generally not a fan of commenting something. We have git so we can revert if needed. Who knows what may be written then.

File: guides/doc-Planning_for_Project/topics/Hosts.adoc [hunk 5]
```diff
@@ -22,17 +22,16 @@ Locations can be used to map the network infrastructure to prevent incorrect hos
...
 It is recommended to configure the majority of settings at the host group level instead of defining hosts directly.
 Configuring a new host then largely becomes a matter of adding it to the right host group.
 As host groups can be nested, you can create a structure that best fits your requirements (see xref:sect-Architecture_Guide-Host_Group_Hierarchies[]).
 
 * *Host collections* – a host registered to {ProjectServer} for the purpose of subscription and content management is called *content host*.
 Content hosts can be organized into host collections, which enables performing bulk actions such as package management or errata installation.
 
-Locations and host groups can be nested, organizations and host collections are flat.
+Locations and host groups can be nested.
+Organizations and host collections are flat.
```
Comment (ekohl) [inline guides/doc-Planning_for_Project/topics/Hosts.adoc:33]: Note orgs can be nested in vanilla Foreman. Katello disables that. It also introduces the concept of host collections, which vanilla Foreman doesn't have.
```suggestion
ifdef::katello,satellite,orcharhino[]
Organizations and host collections are flat.
endif::[]
```

File: guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc (hunk 4 above)
Comment (Lennonka) [reply guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc:261]: Ok, I'm removing it.

File: guides/doc-Planning_for_Project/topics/Hosts.adoc [hunk 6]
```diff
@@ -22,17 +22,21 @@ Locations can be used to map the network infrastructure to prevent incorrect hos
...
+endif::[]
+ifndef::katello,satellite,orcharhino[]
+Organizations, locations, and host groups can be nested.
+endif::[]
```
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/doc-Planning_for_Project/topics/Hosts.adoc:38]: Even better than my suggestion. Thanks!

**Review by ekohl — APPROVED** [review]

**Comment by Lennonka** [comment]
Cherry-picked:
- 056c09a75..f70ef454a  3.4 -> 3.4
- 0928b6044..c48d95a53  3.3 -> 3.3

**Pull request merged**